docs/book/v3/usage.md

```diff
@@ -0,0 +1,84 @@
+# Usage
+
+    <?php
+    
+    declare(strict_types=1);
+    
+    namespace Api\Example\Service;
+    
+    use Dot\UserAgentSniffer\Data\DeviceData;
+    use Dot\UserAgentSniffer\Service\DeviceServiceInterface;
+    
+    /**
+     * Class MyService
+     * @package Api\Example\Service
+     */
+    class MyService
+    {
+        /** @var DeviceServiceInterface $deviceService */
+        protected $deviceService;
+    
+        /**
+         * MyService constructor.
+         * @param DeviceServiceInterface $deviceService
+         */
+        public function __construct(DeviceServiceInterface $deviceService)
+        {
+            $this->deviceService = $deviceService;
+        }
+    
+        /**
+         * @param string $userAgent
+         * @return DeviceData
+         */
+        public function myMethod(string $userAgent)
+        {
+            return $this->deviceService->getDetails($userAgent);
+        }
+    }
+
+When called with an `$userAgent = 'Mozilla/5.0 (iPhone; CPU iPhone OS 13_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) CriOS/78.0.3904.84 Mobile/15E148 Safari/604.1'`, `myMethod($userAgent)` returns an object with the following structure:

```

Comment:
Indent the below sample output as in the [README.md](https://github.com/dotkernel/dot-user-agent-sniffer/blob/3.0/README.md?plain=1#L70) file.